pkg/vpcmodel/nodesExplainability.go

```diff
@@ -269,6 +321,10 @@ func stringExplainabilityLine(c *VPCConfig, connQuery *common.ConnectionSet, src
 	}
 	resStr := ""
 	switch {
+	case router == nil && src.IsExternal():
+		resStr += fmt.Sprintf("%v no fip router and src is external\n", noConnection)
+	case router == nil && dst.IsExternal():
+		resStr += fmt.Sprintf("%v no router (fip/pgw) and dst is external\n", noConnection)
```

Comment:
because in the first case only `fip` can enable the connection. The router in the struct should be `nil` if its not `fip` for an external address.

Reply:
maybe add a comment or explanation in the string itself? (so that it is clear that `pgw` is only relevant for egress connection to public internet destination, as opposed to ingress connections).

Reply:
done; however, I think at does not make sense to spend to much time on the output now - we will finalize after everything else is done, along with other formats. Moreover, now each output change requires manual change to the tests.

Reply:
a comment with indication for later is also ok.

Reply:
never mind, already done. For next time..